src/openforms/conf/locale/nl/LC_MESSAGES/django.po

```diff
@@ -3658,9 +3657,9 @@ msgid ""
 "fill in the following code to retrieve the form submission:<br /><br /> "
 "%(code)s <br /><br />"
 msgstr ""
-"U wordt hierna omgeleid naar een inlogpagina. Nadat u ingelogd bent, kunt "
-"ude de volgende code invoeren om de formulierinzending op te halen:<br /><br"
-" /> %(code)s <br /><br />"
+"Daarna wordt u doorgestuurd naar een pagina waar u inlogt met uw DigiD. "
```

Comment:
```suggestion
"Daarna wordt u doorgestuurd naar een pagina waar u moet inloggen. "
```
We don't know for sure that it's DigiD, technically it's possible you use co-sign with eHerkenning or other authentication plugins (could even be an employee login with OIDC!)